builtin/game/chatcommands.lua

```diff
@@ -315,7 +315,7 @@ core.register_chatcommand("remove_player", {
 
 core.register_chatcommand("teleport", {
 	params = "<X>,<Y>,<Z> | <to_name> | (<name> <X>,<Y>,<Z>) | (<name> <to_name>)",
-	description = "Teleport to player or position",
+	description = "Teleport yourself or player to a position or player",
```

Comment:
Needlessly wordy.

Reply:
Suggest a shorter text without loss of information.

Reply:
The original is fine, but if you really want to specify who the action is on, maybe "Teleport player to ..." The command user is included under this.

Reply:
Yes this seems awkward, maybe 'Teleport player to position or player'?